### ISSUE TITLE
Syntax highlighting in Code Blocks without example

### ISSUE THREAD

**Issue by philipisik** [trulia/hologram#222]
Hi,

is it possible to get html syntax highlighting in a code block without example? Like:

/*doc
\---
title: Title
name: name
category: category
\---

\`\`\`
`<div class="html-example"></div>`
\`\`\`
*/

**Comment by lifeiscontent** [comment]
@philipisik yes you just need to write a custom renderer or only allow your template to output the code example without the preview.

here's a sass code_example_renderer that I've written.

``` ruby
Hologram::CodeExampleRenderer::Factory.define 'sass' do
  example_template 'sass_example_template'
  table_template 'sass_table_template'
  lexer { Rouge::Lexer.find('sass') }
end
```

and then in the templates dir I just provide the `sass_example_template` and the `sass_table_template`
